Mathlib/RepresentationTheory/GroupCohomology/Basic.lean

```diff
@@ -220,6 +220,19 @@ def inhomogeneousCochainsIso : inhomogeneousCochains A ≅ linearYonedaObjResolu
     Category.comp_id]
 #align group_cohomology.inhomogeneous_cochains_iso groupCohomology.inhomogeneousCochainsIso
 
+/-- The `n`-cocycles `Zⁿ(G, A)` of a `k`-linear `G`-representation `A`, i.e. the kernel of the
+`n`th differential in the complex of inhomogeneous cochains. -/
+def cocycles (n : ℕ) : ModuleCat k := (inhomogeneousCochains A).cycles n
```

Comment:
```suggestion
def cocycles (n : ℕ) : ModuleCat k := (inhomogeneousCochains A).cycles n
```
I would rather declare this as an abbreviation, and similarly for the definitions below of the form `(inhomogeneousCochains A).something`. This may help for testing/improving automation for the proofs.